### DISCUSSION TITLE
PDF Template Company details are not displayed in the generated file

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#17186] [How to] [answered]
Hello,

I try to create a pdf template located in the opportunity module in which I need to show several values coming from the Company/Departments module, such as company name, NIF / CIF, and simply the values of the fields are not shown in the generated file.

I tried gitstable and the same thing happens.
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/e54d8693-0cfa-4a5f-8bdb-397f685d43ba)
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/96985476/b420faf7-30af-4711-adeb-af92aef2e7c5)

**Comment by krseguros** [comment]
Greetings,

Can someone explain to me why this happens? I can't get any value from the module relate/Departments/Details/Company name

**Reply by apcloic** [reply, marked as answer]
@krseguros I've made a quick test on Gitstable, I dont't see any issue.
Check in your Departments module (MultiCompany) if the ID is correct (ID 1 on your example), maybe you're trying to get information from another departement.
https://gitstable.yetiforce.com/index.php?module=MultiCompany&view=List&mid=152&parent=76

**Reply by krseguros** [reply]
Indeed, after your response I checked and saw that there were two ID1 and ID2 records, one in the archive and the other in the trash, after I delete these records and leave only the ID1, the company data appears.

Thank you so much!!!
@davide-alghi

**Comment by mariuszkrzaczkowski** [comment]
you can give without ID, then it will load it from the currently logged in user. 
`$(organization : company_name)$`
